### PR TITLE
Move /sshSSOPublicKey to /connectivity/sshSsoPublicKey

### PR DESCRIPTION
**Pull request by marians**
### What this PR does / why we need it

Towards https://github.com/giantswarm/roadmap/issues/2142

### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/test create
/test upgrade

### CODE REVIEW

**Review by calvix — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - Rename /proxy/no_proxy to /connectivity/proxy/noProxy
     - Rename /proxy/http_proxy to /connectivity/proxy/httpProxy
     - Rename /proxy/https_proxy to /connectivity/proxy/httpsProxy
+  - Move /sshSSOPublicKey to /connectivity/sshSsoPublicKey
```
[the remark this answers is not in the document]
Comment (marians) [reply CHANGELOG.md:33]: To me it looks correct. Line 33 should be on the same level as 29. 30 to 32 are sub-items of line 29.

**Comment by marians** [comment]
What has triggered these e2e tests? They are expected to fail. Am I fine to merge?

**Comment by fiunchinho** [comment]
> What has triggered these e2e tests? They are expected to fail. Am I fine to merge?

You did 
image


Yes, it's fine to merge.

**Comment by marians** [comment]
OK. I somehow didn't expect to trigger tests from the main comment. :) Thanks for the hint.

**Pull request merged**